translations/expansion_content.tex/pl.po

```diff
@@ -99,6 +99,13 @@ msgid ""
 "Spells with one School symbol on them are \\textbf{Basic Spells}, while Spells with four identical symbols are \\textbf{Expert Spells}. Certain game effects may affect only one of these types.\n"
 "\n"
 msgstr ""
+"\\subsection*{Szkoła Magii}\n"
+"Wiele rozszerzeń ma efekty, które odnoszą się do Szkół Magii\\index{Schools of Magic}.\n"
+"Wszystkie Karty Zaklęć należą do jednej ze Szkół: Powietrza, Ognia, Ziemi lub Wody.\n"
+"Podczas rzucania \\textbf{Zaklęcia ,,Magiczna Strzała''} musisz wybrać Szkołę, do której należy.\n"
```

Comment:
Proponuje 'Magiczna Strzała'

Reply:
@qwrtln jaka jest Twoja opinia? zgaduje ze to moze tyczyc sie rowniez innych językow